frontend/src/metabase/lib/revisions/revisions.js

```diff
@@ -226,7 +234,8 @@ export function getRevisionEventsForTimeline(revisions = [], canWrite) {
       if (
         !revision.is_creation &&
```

Comment:
Maybe this chain of conditions could be extracted to a const with a declarative name.

Reply:
Good call @gusaiani, thansk. Done ✅